src/trace_processor/importers/common/process_tracker.h

```diff
@@ -113,11 +113,16 @@ class ProcessTracker {
   // Called when a task_newtask without the CLONE_THREAD flag is observed.
   // This force the tracker to start both a new UTID and a new UPID.
   UniquePid StartNewProcess(std::optional<int64_t> timestamp,
-                            std::optional<int64_t> parent_tid,
+                            std::optional<UniquePid> parent_upid,
                             int64_t pid,
                             StringId main_thread_name,
                             ThreadNamePriority priority);
 
+  // Associates a process with its parent thread. Used when the tid that
```

Comment:
I would rephrase both the funciton and comment to be more explicit that this funciton should be used when a thread A "spawns/forks" a thread B .

Reply:
I'd be very explicit. Maybe something like: AddAssociationBetweenCreatedProcessAndParentThread